app/server/toronto-bids/toronto-bids/src/toronto_bids/documents.clj

```diff
@@ -0,0 +1,98 @@
+(ns toronto-bids.documents
+	(:gen-class)
+	(:require [clojure.string :as str])
+	(:require [clojure.java.jdbc :as jdbc])
+	(:require [clojure.data.json :as json])
+)
+
+(def SEARCH-STRING 
+	(str 
+		"document.id IN (SELECT id FROM" 
+		" (SELECT id, short_description, description, search_text FROM document" 
+		" HAVING MATCH(short_description, description, search_text) AGAINST (? IN BOOLEAN MODE)) AS found)"
+	)
+)
+
+; The argument-string
+; (call_number type division commodity commodity_type posting_date_after posting_date_before closing_date_after closing_date_before buyer search-text) 
+
+(def TEST-LIST 
```

Comment:
Rename this function to reflect that it's used to create a list of arguments document fetch query?

Reply:
Not sure what you mean here.